extension/css/cryptup.css

```diff
@@ -1375,6 +1353,23 @@ td {
   opacity: 0.75;
 }
 
+#pgp_block .remote_image_container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: max-content;
```

Comment:
With `width: max-content;` large images cause overflow and add scroll.
Let's replace it with `max-width: 100%`, so all images will fit into page width

Reply:
![image](https://user-images.githubusercontent.com/77344191/223620647-4256b63b-68da-48d4-96bd-6723b08c5b16.png)

If we don't use `width: max-content`, warning text will overflow and look in multiple lines. (In cutomer email. Maybe it's because parent element is a tag though).
Are you fine with it?

Reply:
Ah, got it, then we probably should have both - `width: max-content` and `max-width: 90%`, so warning will fit in 1 line and large images won't have overflow.

Reply:
Sounds good